drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -126,13 +126,24 @@ def self.options_for_type(pick_list_type, user:, relation_id: nil)
         Hmis::File.all_available_tags.map do |tag|
           {
             code: tag.id,
-            label: "#{tag.name} (included: #{tag.included_info})",
+            label: tag.name,
             group_code: tag.group,
             group_label: tag.group,
+            secondary_label: "(includes: #{tag.included_info})",
```

Comment:
secondary label should be empty if there is no `included_info`:
<img width="435" alt="Screen Shot 2023-03-24 at 4 16 48 PM" src="https://user-images.githubusercontent.com/5333982/227630169-0eb430fd-3795-47a9-b7f6-c6973114222d.png">

Reply:
Fixed